### DISCUSSION TITLE
Yazi preview not work except folders

### DISCUSSION THREAD

**Discussion by towry** [sxyazi/yazi#1204] [1. Q&A]
yazi version: Yazi 0.2.5 (VERGEN_IDEMPOTENT_OUTPUT 1980-01-01)

using the latest preset config: https://github.com/sxyazi/yazi/blob/main/yazi-config/preset/yazi.toml

![截屏2024-06-25 11 27 16](https://github.com/sxyazi/yazi/assets/8279858/793e29b2-513e-4b6d-a4f6-714441aaa77d)

![截屏2024-06-25 11 24 16](https://github.com/sxyazi/yazi/assets/8279858/9d4bed96-f520-44d8-bac9-13c1b89674f4)

How to debug this?

**Comment by sxyazi** [comment]
You should use the default config from the `latest` tag unless you're using the latest code of Yazi, see https://yazi-rs.github.io/docs/configuration/overview

**Reply by towry** [reply]
Thanks, the latest tag config works.

> You can find the default configuration files on the latest tag https://github.com/sxyazi/yazi/tree/latest/yazi-config/preset.

I catch this line on second check. 😂
